src/spikeinterface/generation/drifting_generator.py

```diff
@@ -0,0 +1,401 @@
+"""
+This module implements generation of more realistics signal than `spikeinterface.core.generate`
+
+  * drift
+  * correlated noise
+  * far neuron noise
+  * true template or synthetic templates
+
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pathlib import Path
+
+
+import probeinterface
+import probeinterface.plotting
+
+import spikeinterface.full as si
+from spikeinterface.core.generate import generate_unit_locations
+
+from probeinterface import Probe, generate_linear_probe, generate_multi_columns_probe
+from spikeinterface.generation import interpolate_templates, move_dense_templates, DriftingTemplates, make_linear_displacement, InjectDriftingTemplatesRecording
+from spikeinterface.core.generate import generate_templates, generate_unit_locations
+from spikeinterface.core import Templates
+
+from spikeinterface.generation.tests.test_drift_tools import make_some_templates
+
+from probeinterface.plotting import plot_probe
+
+from spikeinterface.generation import interpolate_templates, move_dense_templates, DriftingTemplates
+from spikeinterface.core.generate import generate_templates, default_unit_params_range
+from spikeinterface.core import Templates
+
+
+def make_one_displacement_vector(
+        drift_mode="zigzag",
+        duration=600.,
+        amplitude_factor=1.,
+        displacement_sampling_frequency=5.,
+        t_start_drift=None,
+        t_end_drift=None,
+        period_s=200,
+        # amplitude_um=20.,
+        bump_interval_s=(30, 90.),
+        seed=None
+    ):
+    """
+    Generator a toy discplacement vector like ziagzag or bumps.
+
+    
+    Parameters
+    ----------
+    drift_mode: "zigzag" | "bumps"
```

Comment:
I'm assuming these are the technical terms for drift :P

Reply:
This is the naing we chosse for the paper in eneuro. See https://www.eneuro.org/content/11/2/ENEURO.0229-23.2023